tools/cli/test/e2e/cli/split_test.go

```diff
@@ -14,8 +14,8 @@ import (
 )
 
 var versions = []string{
-	"2023-01-01",
-	"2023-02-01",
+	// "2023-01-01",
```

Comment:
[need a follow-up PR]

The 2023-01-01 clusters endpoints dont show as deprecated in the current specs, and our logic is marking them as deprecated rightfully. Need to check w/ the team if we want to fix this os ir if was a desired behaviour. 

All other items are matching.